src/read-documents/read-documents.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/sequelize';
+import { UsersService } from 'src/users/users.service';
+import { ReadDocuments } from './models';
+
+@Injectable()
+export class ReadDocumentsService {
+  constructor(
+    @InjectModel(ReadDocuments)
+    private readDocuments: typeof ReadDocuments,
+    private usersService: UsersService
+  ) {}
+
+  async findOneUser(userId: string) {
+    const user = await this.usersService.findOne(userId);
+    return user;
+  }
+
+  async createReadDocument(UserId: string, documentName: string) {
+    const readDocument = await this.readDocuments.create({
+      UserId,
+      documentName,
+    });
+    return readDocument;
```

Comment:
Pareil, tu peux directement retourner : ` return this.readDocuments.create({
      UserId,
      documentName,
    });`